File: LeetCode-Programs/Largest Number At Least Twice of Others.cpp
class Solution {
public:
    int dominantIndex(vector<int>& nums) { 
        int a=*max_element(nums.begin(),nums.end()), ans;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]==a) 
            {
                ans=i; 
                continue;
            }
            if(nums[i]*2>a) 
                return -1;
        }
        return ans;
    }
};

File: LeetCode-Programs/Largest Number At Least Twice of Others.py
class Solution:
    def dominantIndex(self, nums: List[int]) -> int:
        a=max(nums)
        for i in range(len(nums)):
            if nums[i]==a:
                ans=i
                continue
            if nums[i]*2>a:
                return -1
        return ans
        
